jsonifier/models/models.py

```diff
@@ -256,3 +266,7 @@ def _jsonify_format_duration(self, fname):
 
         """
         return format_duration(self[fname])
+
+    def default_function(self, field):
```

Comment:
After the refactor suggested above, this should probably also be prefixed with `_`, and probably also with `_jsonify` in order not to pollute the namespace.

Reply:
initial solution was best. now a sister key is there no more function -meddling.